### ISSUE TITLE
Docs Issue: Flow.edges_from() and edges_to()

### ISSUE THREAD

**Issue by shaunc** [PrefectHQ/prefect#3032]
They are documented as:
```
Returns:
dict with the key as the task passed in and the value as a set of all edges leading from that task
```
However their actual signature is not a dictionary, but rather:
```python
    def edges_from(self, task: Task) -> Set[Edge]:
```
(which makes sense, as why pass task and then also index by task to get the edges you requested.)

**Comment by joshmeek** [comment]
Thanks for pointing this out @shaunc!

**Comment by andykawabata** [comment]
First time open-source contributor here. I would like to fix this issue, and just want to clarify that instead of saying:
```
Returns:
dict with the key as the task passed in and the value as a set of all edges leading from that task
```
it should say:
```
Returns:
Set of all edges leading from that task
```
is this correct?

**Comment by cicdw** [comment]
Hi @andykawabata and welcome!  Yes that would work; to better fit with our other doc Return signatures (which are of the form `- type: description` you might also do:
```python
- Set: set of all edges leading from that task
```
but it's not super important either way.  Also note that this needs updating on both `flow.edges_from` and `flow.edges_to` :+1:

**Comment by shaunc** [comment]
Thanks @andykawabata ! You're encouraging me by example. :)

**Comment by cicdw** [comment]
Closed by #3128
